data/actions/scripts/valuables/unrealizedDream.lua

```diff
@@ -0,0 +1,105 @@
+function onUse(player, item, fromPosition, target, toPosition, isHotkey)
```

Comment:
Nice.

I hope saying this won't demotivate you, 
but chances are not correct and shouldn't the player himself go to the dreamcatcher device?

http://tibia.wikia.com/wiki/Unrealized_Dream/Statistics